english/count-the-marshmellows/sell-on-ebay/running-delivery/turn-super-saiyan/investigate-ki/investigate-ki.md

```diff
@@ -0,0 +1,25 @@
+You find Vegeta (The source of the large ki) eating marshmallows.
+
+These marshmallows are numerous and they look so delicious as well.
+
+So you defeat Vegeta and take control of all the yummy marshmallows.
+
+What do you do?
+
+
+[dig a hole with your bare hands?](../../../../../hands/hands.md)
+
+[Invite some friends over?](../../../../../invite-friends/friends.md)
+
+[Count the marshmallows?](../../../../../count-the-marshmellows/count-the-marshmellows.md)
+
+[Begin to dance to pass the time?](../../../../../dance/dance.md)
+
+[Make one of those weird marshmallow Jell-O salads?](../../../../../marshmallow-jello-o-salad/the-kind-your-grandma-used-to-make.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.